build/includes/examples.mk

```
@@ -42,8 +42,9 @@ push-example-golang-images: example-image-push.supertuxkart
 push-example-golang-images: example-image-push.simple-game-server
 
 # Test to ensure the example image found in the % folder is on GAR. Fails if it is not.
-example-image-test.%:
-	$(DOCKER_RUN) bash -c "cd examples/$* && make gar-check"
+example-image-test.%: 
+	$(DOCKER_RUN) bash -c "cd examples/$* && make gar-check" $(DOCKER_RUN_ARGS)
```

```suggestion
	$(DOCKER_RUN) bash -c "cd examples/$* && make gar-check"
```

🤔  do you need this change? I would have thought the $(DOCKER_RUN_ARGS) would be included in the `$(DOCKER_RUN)` command?

---

Thanks for the review. I have updated this line.

---

Just confirming, you tested it, and it works without the updated line?

---

Yes, I have tested with agones-mangalpalli. Please find the build log - https://paste.googleplex.com/5228291551133696

---

Awesome. Thanks for confirming!